pages/events/Gray Foundation Annual Summit.mdx

```diff
@@ -0,0 +1,5 @@
+
+
+# Gray Foundation Annual Summit
```

Comment:
This is actually a pretty exclusive event with the Grays where teams mainly cover unpublished/in-progress research, so it's not public, but I wonder if we can try to get group photos from the last one/upcoming one. I think we can put the date of the last summit (Jun 1st 2022) and something like **2023 Summit date TBD**